models/stg_recharge__product.sql

```diff
@@ -2,7 +2,7 @@
 with base as (
 
     select *
-    from {{ ref('stg_recharge__product_tmp') }}
+    from {{ ref('stg_recharge__product_tmp') }} 
```

Comment:
Just a small note that this will likely change to plan

Reply:
I have updated the table names, same thing with the column names.